ReactAndroid/src/main/java/com/facebook/react/bridge/queue/MessageQueueThreadImpl.java

```diff
@@ -184,27 +184,8 @@ private static MessageQueueThreadImpl startNewBackgroundThread(
     final String name,
     long stackSize,
     QueueThreadExceptionHandler exceptionHandler) {
-    final SimpleSettableFuture<Looper> looperFuture = new SimpleSettableFuture<>();
-    final SimpleSettableFuture<MessageQueueThread> mqtFuture = new SimpleSettableFuture<>();
-    Thread bgThread = new Thread(null,
-        new Runnable() {
-          @Override
-          public void run() {
-            Process.setThreadPriority(Process.THREAD_PRIORITY_DISPLAY);
-            Looper.prepare();
-
-            looperFuture.set(Looper.myLooper());
-            MessageQueueThreadRegistry.register(mqtFuture.getOrThrow());
```

Comment:
Well, we should do another change first then to clean that up to make sure that's true.